rpc/ethereum/backend/backend.go

```diff
@@ -1017,55 +1016,6 @@ func (e *EVMBackend) BaseFee(height int64) (*big.Int, error) {
 	return nil, nil
 }
 
-// GetFilteredBlocks returns the block height list match the given bloom filters.
-func (e *EVMBackend) GetFilteredBlocks(
```

Comment:
@JayT106 can you comment on this change? I think you added this logic so want to make sure you are aware of it being deleted

Reply:
yeah, I find this function seems not much useful, the current iterate logic is like this:
```
if req.block_hash != nil {
    // single block
    return blockLogs(bloom, height)
} else {
    for height=start; height< end; height++ {
        logs := append(logs, blockLogs(bloom, height))
    }
}

func blockLogs(bloom, height) {
    if !bloomFilter(bloom, filterCritiria) {
        // fast path
        return
    }
    get logs from block and filter
}
```